addons/event_booth_sale/tests/test_event_booth_sale.py

```diff
@@ -117,6 +121,42 @@ def test_event_booth_prices_with_sale_order(self):
                 booth.state, 'unavailable',
                 "Booth should not be available anymore.")
 
+    @users('user_sales_salesman')
+    def test_event_booth_registrations_inverse(self):
+        # Create sale order and add a booth
+        sale_order = self.env['sale.order'].create({
+            'partner_id': self.event_customer.id,
+            'pricelist_id': self.test_pricelist.id,
+            'order_line': [
+                Command.create({
+                    'product_id': self.event_booth_product.id,
+                    'event_id': self.event_0.id,
+                    'event_booth_category_id': self.event_booth_category_1.id,
+                    'event_booth_pending_ids': (self.booth_1 + self.booth_2).ids
+                })
+            ]
+        })
+
+        self.assertEqual(sale_order.order_line.event_booth_registration_ids.event_booth_id.ids,
+                         (self.booth_1 + self.booth_2).ids,
+                         "Booths not correctly linked with event_booth_registration.")
+
+        # Update booths
+        sale_order.write({
+            'order_line': [
+                Command.update(sale_order.order_line.id, {
+                    'event_booth_pending_ids': [Command.set(self.booth_3.ids)]
```

Comment:
Can we select 2 and 3 similarly to what was done in the original commit? I feel like it's a bit more coverage.